nservicebus/upgrades/release-policy.md

```diff
@@ -41,9 +42,9 @@ Some examples:
  * Version 4.6 was released on 2014-05-01 this means that its latest patch release will be supported till 2015-05-01. This means that there will be no fixes minor versions after 2015-05-01. It is required to update to at least a newer minor version that is still supported as this version will receive a patch release.
  * A newer patch release will automatically mean that the previous patch release will be obsolete. Particular will apply a bugfix on the latest patch release but will not officially release a patch for a obsolete patch release. In other words, version X.Y.3 will not be patched (to version X.Y.3.1) when version X.Y.4 is the latest patch. That would then be released as version X.Y.5
 
-It is strongly recommend to upgrade frequently enough to stay on a supported version. For a best upgrade experience upgrade from major version to major version. For example if using NServiceBus Version 3.3.15 and want to upgrade to NServiceBus Version 5.2.3. First upgrade to the latest NServiceBus Version 4.x release (i.e. Version 4.7.6). Follow the suggested API upgrade guides and obsoletion messages and then upgrade to NServiceBus Version 5.2.3.
+It is strongly recommend to upgrade frequently enough to stay on a supported version. For a best upgrade experience upgrade from major version to major version. For example if using NServiceBus Version 3.3.15 and want to upgrade to NServiceBus Version 5.2.3. First upgrade to the latest NServiceBus Version 4.x release (i.e. Version 4.7.6). Follow the suggested API upgrade guides and [ObsoleteAttribute messages](https://msdn.microsoft.com/en-us/library/system.obsoleteattribute.aspx) and then upgrade to NServiceBus Version 5.2.3.
 
-[Contact support](https://particular.net/support) if there are any bugfixes that believe should be back-ported to a specific. version
+[Contact support](https://particular.net/support) if there are any bugfixes that believe should be back-ported to a specific version.
```

Comment:
>[Contact support](https://particular.net/support) if there are any bugfixes that believe should be back-ported to a specific version.

How about:
[Contact support](https://particular.net/support) if there are any bugfixes that needs to be back-ported to a specific version.